evansmusomi/react-native-102
# Handles Deletion of Places and Showing Newly Added Places

*What this does*
- Handles deleting places on the server and client
- Displays newly added places on the client once it's saved on the server

*How to verify*
- Deleting a place should remove it from the Find Places list and Firebase
- Sharing a place should make it immediately visible on the Find Places list
